src/cascadia/TerminalApp/Tab.h

```diff
@@ -88,6 +87,8 @@ namespace winrt::TerminalApp::implementation
         // The TabViewNumTabs is the number of Tab objects in TerminalPage's _tabs vector.
         OBSERVABLE_GETSET_PROPERTY(uint32_t, TabViewNumTabs, _PropertyChangedHandlers, 0);
 
+        OBSERVABLE_GETSET_PROPERTY(winrt::Windows::UI::Xaml::UIElement, Content, _PropertyChangedHandlers, nullptr);

```

Comment:
/cc @leonmsft this will conflict with you i bet

Reply:
it probably will, but at least I named the variable `Content` as well 😄, i think the merge conflict won't be too bad for this property, it'll probably be bad in terms of me literally deleting `Tab.cpp` and replacing it with `TerminalTab.cpp`. though if you merge this PR first I'd be willing to resolve those conflicts 👍